apps/foundation-demo/src/app/core/footer/footer.module.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { FooterModule } from './footer.module';
+
+describe('FooterModule', () => {
```

Comment:
I don't think we need this. The build will just fail and the app won't work if the imports are wrong

Reply:
@lderrickable I'd agree. But I'd also like to understand why it's now auto generated by `ng g m`